frontend/apps/filemanager/filemanagerconverter.lua

```diff
@@ -0,0 +1,104 @@
+--[[--
+This module is responsible for converting files.
+]]
+
+local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
+local ConfirmBox = require("ui/widget/confirmbox")
+local UIManager = require("ui/uimanager")
+local lfs = require("libs/libkoreader-lfs")
+local logger = require("logger")
+local util = require("util")
+local gettext = require("gettext")
+local T = require("ffi/util").template
+
+local FileConverter = {
+    formats_from = {
+        md = {
+            name = gettext("Markdown"),
+            from = "markdown",
+        },
+    },
+    formats_to = {
+        epub = {
+            to = "epub",
+        },
+        html = {
+            to = "html",
+        },
+        pdf = {
+            to = "pdf",
+        },
+    },
+    --curl --form input_files[]=@README.md --form from=markdown  --form to=pdf http://c.docverter.com/convert
+    docverter_url = "http://c.docverter.com/convert",
+}
+
+--- Converts a markdown fragment to a full HTML document.
+---- @string markdown the markdown fragment
+---- @string title an optional title for the HTML document
+---- @treturn string an HTML document
+function FileConverter:_mdToHtml(markdown, title)
```

Comment:
If you plan to make this a public method that will be used by other modules, better not to prefix it with `_`.